rpcserver.go

```diff
@@ -4218,12 +4219,32 @@ func marshalUnsealedSeedling(verbose bool,
 		if err != nil {
 			return nil, err
 		}
+
+		// Generate PSBT equivalent of the group virtual tx.
+		groupVirtualPsbt, err := psbt.NewFromUnsignedTx(
```

Comment:
This doesn't really give us anything... This can trivially also be `bitcoin-cli converttopsbt <tx>`.
If we want to give a signer any chance of being able to sign this, we need to add the minimum amount of metadata we have available:

```
		groupVirtualPsbt.Inputs[0].WitnessUtxo = &wire.TxOut{
			Value:    0, // Probably needs to be set to the equal amount as the VM TX's output value?
			PkScript: nil, // Needs to be the p2tr script for the script key Taproot key.
		}
		groupVirtualPsbt.Inputs[0].TaprootInternalKey = ? // I think we can set this from the script key's internal key, which I assume we need to know at this point?
		groupVirtualPsbt.Inputs[0].TaprootMerkleRoot = ? // Same as above, should be the tweak of the script key.

		groupVirtualPsbt.Outputs[0].TaprootInternalKey = ? // Same as above.
```